### PR TITLE
Fix/inspector calculated values

### PR DESCRIPTION
**Pull request by enidemi**
Fixing inspector hook `useInspectorLayoutInfo` a little for showing calculated values.

The issue was that in one part is was using specialSizeMeasurements for padding only and added a special propertyStatus to have different colors, the other part was using calculated style but didn't set the propertyStatus for the colors. The specialSizeMeasurements is also limited to padding and margin only, so it's not a good fit here.

Changed the text color to grey from light blue for these detected values.

### CODE REVIEW

File: editor/src/components/inspector/common/property-path-hooks.ts [hunk 1]
```diff
@@ -376,31 +375,49 @@ function parseFinalValue<PropertiesToControl extends ParsedPropertiesKeys>(
...
+  } {
+    const parsedRealValue = parseAnyParseableValue(property, realValue, null)
+    if (isRight(parsedRealValue)) {
+      return {
+        finalValue: parsedRealValue.value,
+        isUnknown: isUnknown,
+        usesComputedFallback: false,
+      }
     }
+    const parsedComputedValue = parseAnyParseableValue(property, computedValue, null)
```
Comment (Rheeseyb) [inline editor/src/components/inspector/common/property-path-hooks.ts:398]: Can you turn these into `else if` statements please so it's immediately clear to see where the logic forks

File: editor/src/components/inspector/common/property-path-hooks.ts [hunk 2]
```diff
@@ -768,60 +788,9 @@ export function useInspectorLayoutInfo<P extends LayoutProp | StyleLayoutProp>(
...
 
-function getValueFromSpecialSizeMeasurements<P extends LayoutProp | StyleLayoutProp>(
-  property: P,
-  specialSizeMeasurements: SpecialSizeMeasurements,
-): ParsedProperties[P] | null {
-  // TODO: are there other properties in special size measurementes to extract?
-  let value: number | undefined = undefined
-  switch (property) {
-    case 'paddingLeft':
-      value = specialSizeMeasurements.padding.left
```
Comment (balazsbajorics) [inline editor/src/components/inspector/common/property-path-hooks.ts:794]: do we use specialSizeMeasurements.padding and specialSizeMeasurements.margin anywhere else? if not, we could remove them. but maybe some canvas controls use them?

File: editor/src/components/inspector/common/property-path-hooks.ts [hunk 3]
```diff
@@ -376,31 +375,49 @@ function parseFinalValue<PropertiesToControl extends ParsedPropertiesKeys>(
...
   const rawValueAsMaybe = eitherToMaybe(rawValue)
 
-  function finalValueFromReal(): ParsedPropertiesValues {
-    if (realValue == null) {
-      const parsedComputedValue = parseAnyParseableValue(property, computedValue, null)
-      return defaultEither(emptyValues[property], parsedComputedValue)
-    } else {
-      const parsedRealValue = parseAnyParseableValue(property, realValue, null)
-      return defaultEither(emptyValues[property], parsedRealValue)
+  function finalValueFromReal(
```
Comment (balazsbajorics) [inline editor/src/components/inspector/common/property-path-hooks.ts:383]: this is very hard to read :|

File: editor/src/components/inspector/common/property-path-hooks.ts (hunk 2 above)
Comment (Rheeseyb) [reply editor/src/components/inspector/common/property-path-hooks.ts:794]: We have `margin-controls.tsx` and `padding-controls.tsx` that use them

File: editor/src/components/inspector/common/property-path-hooks.ts (hunk 3 above)
Comment (Rheeseyb) [reply editor/src/components/inspector/common/property-path-hooks.ts:383]: I think extracting the return type would help since it's used above too

Comment (enidemi) [reply editor/src/components/inspector/common/property-path-hooks.ts]: fixed in effda55

**Review by seanparsons — APPROVED** [review]

**Review by balazsbajorics — APPROVED** [review]

**Pull request merged**